apps/web/src/server/lib/nunjucks-globals/get-web-env-config.js

```diff
@@ -0,0 +1,6 @@
+import config from '../../../../environment/config.js';
+
+export const webEnvConfig = {
+	env: config.env,
```

Comment:
FYI If I follow the infra and config through correctly this will be `production` for all Azure environments.

Reply:
also it's probably worth adding a `isTraining` check as we do for the other envs in `base-config.js`

Reply:
Will environment.NODE_ENV be set to 'training' when on https://back-office-training.planninginspectorate.gov.uk/?

Reply:
Nope, `NODE_ENV` will always be `production`. We'll need to add a new env var for this if we need the environment name.

Reply:
we could use APP_HOSTNAME instead of adding a new one? - it's unlikely to change

Reply:
@francescobrenci-AR @beejjacobs It appears APP_HOSTNAME isn't present on all env and as mentioned above NODE_ENV is set to production for all Azure environments. So I've taken a bit of a different approach to determine the env and used hostname in a new middleware function that is applied to all routes. Let me know your thoughts on this approach.